SDK v2 migration guide.md

```diff
@@ -198,6 +198,7 @@ Find a client you currently use below, read the table of API name changes and us
 | `Message.CreationTimeUtc` | `TelemetryMessage.CreatedOnUtc`, `IncomingMessage.CreatedOnUtc` | Conforming to the naming guidelines by the Azure SDK team, where DateTime/Offset types have an "On" suffix (and "Utc" suffix when explicitly in UTC).³ |
 | `Message.EnqueuedTimeUtc` | `TelemetryMessage.EnqueuedtimeUtc`, `IncomingMessage.EnqueuedTimeUtc` | See³ |
 | `Message.ExpiryTimeUtc` | `TelemetryMessage.ExpiresOnUtc`, `IncomingMessage.ExpiresOnUtc` | See³ |
+| `Message.GetBytes` | `IncomingMessage.GetPayloadAsBytes` | See³ |
```

Comment:
This change has nothing to do with Azure SDK naming guidelines.

I'd file it under "More descriptive name."